sdk/js/src/token_bridge/__tests__/helpers.ts

```diff
@@ -92,3 +101,90 @@ export async function transferFromEthToSolana(): Promise<string> {
   provider.destroy();
   return sequence;
 }
+
+export async function waitForTerraExecution(
+  transaction: string
+): Promise<TxInfo | undefined> {
+  const lcd = new LCDClient(TERRA_HOST);
+  let done: boolean = false;
+  let info;
+  while (!done) {
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+    try {
+      info = await lcd.tx.txInfo(transaction);
+      if (info) {
+        done = true;
+      }
+    } catch (e) {
+      console.error(e);
+    }
+  }
+  if (info && info.code !== 0) {
+    // error code
+    throw new Error(
+      `Tx ${transaction}: error code ${info.code}: ${info.raw_log}`
+    );
+  }
+  return info;
+}
+
+export async function getSignedVAABySequence(
+  chainId: ChainId,
+  sequence: string,
+  emitterAddress: string
+): Promise<Uint8Array> {
+  //Note, if handed a sequence which doesn't exist or was skipped for consensus this will retry until the timeout.
+  // console.log("about to do signed vaa with retry");
```

Comment:
let over comments, might want to remove.

Reply:
Removed.